src/main/resources/edu/cornell/kfs/coa/document/workflow/SubObjectCodeGlobalMaintenanceDocument.xml

```diff
@@ -14,26 +14,25 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-<data xmlns="ns:workflow" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
   <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
     <documentType>
       <name>GSOB</name>
       <parent>COA</parent>
+      <label>Sub-Object Code Global</label>
```

Comment:
Yes, that's correct. This is a case where the workflow xml in our project was out of sync with what we had ingested and what's in the database. Global Account Reversion (GARV) was previously changed to have a parent of COSM with KFSPTS-5185, I was just updating the workflow xml in our project with this commit.